mirdata/datasets/phenicx_anechoic.py

```diff
@@ -0,0 +1,413 @@
+            audio_mix += audio
+        return audio_mix,sr
+
+    @core.cached_property
+    def notes(self) -> Optional[annotations.NoteData]:
+        """the track's notes corresponding to the score aligned to the audio
+
+        Returns:
+            NoteData: Note data for the track
+
+        """
+        return load_score(self.notes_path)
+
+    @core.cached_property
+    def notes_original(self) -> Optional[annotations.NoteData]:
+        """the track's notes corresponding to the original score
+
+        Returns:
+            NoteData: Note data for the track
+
+        """
+        return load_score(self.notes_original_path)
+
+    def get_audio_voice(self,id_voice) -> Optional[Tuple[np.ndarray, float]]:
+        """the track's audio
+
+        Returns:
+            * np.ndarray - the mono audio signal
+            * float - The sample rate of the audio file
+
+        """
+        return load_audio(self.audio_paths[id_voice])
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_paths[0],
+            note_data=[(self.notes, "aligned notes")]
+        )
+
+
+
+class MultiTrack(core.MultiTrack):
+    """Phenicx-Anechoic MultiTrack class
+
+    Args:
+        mtrack_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets/Phenicx-Anechoic`
+
+
+    Attributes:
+        audio_path (str): path to the audio files associated with this track
+        annotation_path (str): path to the annotations files associated with this track
+        mtrack_id (str): multitrack id
+        tracks (list): list of track ids within this multitrack
+    """
+    def __init__(
+        self,
+        mtrack_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata=None,
+    ):
+        super().__init__(
+            mtrack_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        #### parse the keys for the dictionary of instruments and strings
+        self.instruments = {
+                source.replace(self.mtrack_id+'-', ''):source
+                for source in self.track_ids
+            }
+        self.sections = {'brass':[],'strings':[],'woodwinds':[]}
+        for instrument,track_id in self.instruments.items():
+            self.sections[DATASET_SECTIONS[instrument]].append(track_id)
+
+        self.piece = self.mtrack_id
+
+    @property
+    def tracks(self):
+        #### this property is redefined here to make use of the Track object defined above
+        return {t: Track(t, self._data_home, self._dataset_name, self._index, self._metadata) for t in self.track_ids}
+
+    @property
+    def track_audio_property(self):
```

Comment:
without doing any other tricks, this was the only way to set this attribute for my class

Reply:
yes! this is how I intended for it to be set. Maybe it needs to be more clear e.g. in contributing?